evaluator/src/main/kotlin/Rule.kt

```diff
@@ -102,26 +102,24 @@ abstract class Rule(
      */
     abstract fun issueSource(): String
 
+    fun issue(severity: Severity, message: String) {
```

Comment:
Should the `message` maybe go first, and `severity` maybe have a default of `ERROR` (like it has in `OrtIssue`)?

Reply:
> Should the `message` maybe go first, 

I prefer the severity first, because it is always short, the message can get long and make the severity argument hard to spot.

>and `severity` maybe have a default of `ERROR` (like it has in `OrtIssue`)?

For this the `error()` function already exists.